client/my-sites/site-indicator/index.jsx

```diff
@@ -205,17 +205,22 @@ class SiteIndicator extends Component {
 
 		// Don't show the button if the site is not defined.
 		if ( site ) {
+			const { siteIsAutomatedTransfer } = this.props;
 			accessFailedMessage = (
 				<span>
 					{ translate( 'This site cannot be accessed.' ) }
 					<Button
 						borderless
 						compact
 						scary
-						href={ `/settings/disconnect-site/${ site.slug }` }
+						href={ `/settings/disconnect-site/${ site.slug }${
+							siteIsAutomatedTransfer ? '' : '?type=down'
+						}` }
```

Comment:
The `/settings/disconnect-site/:site` route won't do anything useful for an Atomic site. It will [redirect to `/settings/general`](https://github.com/Automattic/wp-calypso/blob/909abf45b46a7931d4b54826de11b0eae37a93bd/client/my-sites/site-settings/redirect-non-jetpack/index.jsx#L35-L37).

Reply:
fixed in bececfbe0b8812be9a79c3d8897afe02e783c6c2